src/plugin/plugin-manager.js

```diff
@@ -105,6 +105,15 @@ export default class PluginManager {
     return false;
   }
 
+  /**
+   * Iterates over all the plugins and calls loadMedia().
+   * @public
+   * @returns {void}
+   */
+  loadMedia(): void {
+    Object.keys(this._plugins).forEach(k => this._plugins[k].loadMedia());
+  }
```

Comment:
why creating an array and then iterating and not just loop the object?

Reply:
I think when we started doing object iterations the benchmark showed this is better, but I see now that `for..in` is now very fast: https://hackernoon.com/5-techniques-to-iterate-over-javascript-object-entries-and-their-performance-6602dcb708a8